testing/tests/DevExpress.ui.widgets.form/form.tests.js

```diff
@@ -97,6 +104,160 @@ QUnit.testInActiveWindow('Form\'s inputs saves value on refresh', function(asser
     assert.deepEqual(formData, { name: 'test' }, 'value updates');
 });
 
+
+['xs', 'sm', 'md', 'lg'].forEach(screenSize => {
+    function createForm(testConfig) {
+        const config = extend({
+            colCountByScreen: { xs: 1, sm: 2, md: 3, lg: 4 },
+            items: [
+                { dataField: 'field1' }, { dataField: 'field2' }, { dataField: 'field3' }, { dataField: 'field4' }
+            ],
+        }, testConfig);
+        return $('#form').dxForm(config).dxForm('instance');
+    }
+
+    function getExpectedCallCount(size) {
+        const renderCount = { xs: 21, sm: 23, md: 25, lg: 22 };
+        return renderCount[size];
+    }
+
+    function getColsCount($form) {
+        let result = -1;
+
+        const $lastCol = $form.find(`.${LAST_COL_CLASS}`);
+        [1, 2, 3, 4].forEach(colCount => {
+            if($lastCol.hasClass(`dx-col-${colCount - 1}`)) {
+                result = colCount;
+            }
+        });
+
+        return result;
+    }
+
+    function getRowsCount($form) {
+        return $form.find(`> .${BOX_CLASS} > .${BOX_ITEM_CLASS}`).length;
+    }
+
+    function checkForm(assert, form) {
+        const $form = form.$element();
+        const $formBox = $form.find(`.${RESPONSIVE_BOX_CLASS}`);
+        assert.ok($formBox.hasClass(`dx-responsivebox-screen-${screenSize}`), 'form box has valid class');
```

Comment:
Why do we need to check css class for the current changes in sources?
It looks as a private part of the form internal implementation.